drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -36,8 +36,10 @@
 #include "core/templates/hashfuncs.h"
 #include "drivers/vulkan/vulkan_context.h"
 
+#include "core/io/compression.h"
+#include "core/io/marshalls.h"
+#include "thirdparty/misc/smolv.h"
```

Comment:
Try to keep includes following our style guide: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

So this should be:
```
#include "rendering_device_vulkan.h"

#include "core/config/project_settings.h"
#include "core/io/compression.h"
#include "core/io/file_access.h"
#include "core/io/marshalls.h"
#include "core/os/os.h"
#include "core/templates/hashfuncs.h"
#include "drivers/vulkan/vulkan_context.h"

#include "thirdparty/misc/smolv.h"
#include "thirdparty/spirv-reflect/spirv_reflect.h"

//#define FORCE_FULL_BARRIER
```